package.json

```diff
@@ -4,17 +4,22 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "agent": "ts-node src/agent/main.ts",
+    "chatbot": "ts-node src/chatbot/main.ts"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "baserun": "^1.3.9",
+    "baserun": "github:baserun-ai/baserun-js",
```

Comment:
our fixes to baserun-js are not released to npm registry yet

Reply:
Let's just use the latest released version, which is `^2.0.4`.

Reply:
but there's a bug there that doesn't allow you to annotate a trace. python example does that so I wanted to do it here as well